<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassicSkinSettings</class>
 <widget class="QWidget" name="ClassicSkinSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="orientation_label">
       <property name="text">
        <string>&amp;orientation</string>
       </property>
       <property name="buddy">
        <cstring>orientation_cbox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orientation_cbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="spacing_label">
       <property name="text">
        <string>&amp;spacing</string>
       </property>
       <property name="buddy">
        <cstring>spacing_edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spacing_edit">
       <property name="minimum">
        <number>-8</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="time_format_label">
       <property name="text">
        <string>&amp;time format</string>
       </property>
       <property name="buddy">
        <cstring>time_format_cbox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="time_format_cbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string notr="true">h:mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">hh:mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">hh:mm:ss</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">h:mmA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">h:mm a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">hh:mmA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">hh:mm a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">hh:mm:ssA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">hh:mm:ss a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">H:mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">HH:mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">HH:mm:ss</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="allow_format_edit_cb">
       <property name="text">
        <string>allow &amp;edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply_format_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>249</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
